prime-react/src/main/scala/react/primereact/RadioButton.scala

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.primereact
+
+import cats.Eq
+import cats.Id
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import react.common.*
+import reactST.primereact.components.{RadioButton => CRadioButton}
+
+import scalajs.js
+
+case class RadioButton[A](
+  value:          A,
+  id:             js.UndefOr[String] = js.undefined,
+  inputId:        js.UndefOr[String] = js.undefined, // id of the input element
+  name:           js.UndefOr[String] = js.undefined, // Name of the radio button
+  disabled:       js.UndefOr[Boolean] = js.undefined,
+  clazz:          js.UndefOr[Css] = js.undefined,
+  checked:        js.UndefOr[Boolean] = js.undefined,
+  required:       js.UndefOr[Boolean] = js.undefined,
+  onChange:       js.UndefOr[(A, Boolean) => Callback] = js.undefined,
+  modifiers:      Seq[TagMod] = Seq.empty
+)(using val eqAA: Eq[A])
+    extends ReactFnProps[RadioButtonBase](RadioButtonBase.component)
+    with RadioButtonBase {
+  type AA    = A
+  type GG[X] = Id[X]
```

Comment:
Do we need these types? And the `RadioButtonBase`? Controls like `DropDown` and  `SelectButton` have these because there are also `Optional` and other variants of them.

Reply:
Value is generic here and it seems like I needed the same construct, but I could try to make the component use an Any?